Assignment 2/180501105/attendanceStudents.py

```diff
@@ -0,0 +1,6 @@
+attendance=['Ram-P','Malar-A','Ahuja-P','Vijay-P','Harini-A']
+#print("Students present")
+for a in attendance:
+    if a[-1]=='P':
+        print(a)
+
```

Comment:
The program must be more dynamic to accept any set of names and their attendance. Feel free to include input lines that will help you make this more dynamic ( for example reading the number of students ). Also, we want to just print the names of the students so find a way to do that.

Reply:
changes made